workitem/workitem_repository.go

```diff
@@ -166,10 +288,19 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, typeID string, fiel
 	if err != nil {
 		return nil, errors.NewBadParameterError("type", typeID)
 	}
+
+	// The order of workitems are spaced by a factor of 1000.
+	pos, err := r.LoadHighestOrder()
+	if err != nil {
+		return nil, errors.NewInternalError(err.Error())
+	}
+	pos = pos + 1000
```

Comment:
if 1000 is so magical why not have a constant or variable for this so it is guaranteed to be consistent ?